plugin/zoom/lib/zoom_plugin.class.php

```diff
@@ -0,0 +1,348 @@
+<?php
+
+/* For licensing terms, see /license.txt */
+
+use Chamilo\PluginBundle\Zoom\API\JWTClient;
+use Chamilo\PluginBundle\Zoom\API\ParticipantListItem;
+use Chamilo\PluginBundle\Zoom\API\RecordingMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeeting;
+use Chamilo\PluginBundle\Zoom\CourseMeetingInfoGet;
+use Chamilo\PluginBundle\Zoom\CourseMeetingListItem;
+
+class ZoomPlugin extends Plugin
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class ZoomPlugin has 16 non-getter- and setter-methods. Consider refactoring ZoomPlugin to keep number of methods under 10.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)